docs/api-reference/README.md

```diff
@@ -8,6 +8,7 @@ Use the following reference docs to understand the kubernetes REST API for vario
 * extensions/v1beta1: [operations](/docs/api-reference/extensions/v1beta1/operations.html), [model definitions](/docs/api-reference/extensions/v1beta1/definitions.html)
 * batch/v1: [operations](/docs/api-reference/batch/v1/operations.html), [model definitions](/docs/api-reference/batch/v1/definitions.html)
 * autoscaling/v1: [operations](/docs/api-reference/autoscaling/v1/operations.html), [model definitions](/docs/api-reference/autoscaling/v1/definitions.html)
+* apps/v1beta1: [operations](/docs/api-reference/apps/v1beta1/operations.html), [model definitions](/docs/api-reference/apps/v1beta1/definitions.html)
```

Comment:
@mbohlool any idea why batch/v2alpha1 doesn't show me cronjobs and jobs? http://kubernetes.io/docs/api-reference/batch/v2alpha1/definitions/